Automattic/jetpack
# VideoPress: start to handle loading state for local videos

follow-up of https://github.com/Automattic/jetpack/pull/26794

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: start to handle loading state for local videos

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






*Test with a site with local videos
* Go to the VideoPress dashboard
* Change the page
* Confirm the pagination gets blocked until the async request finished


https://user-images.githubusercontent.com/77539/195422967-c18d6092-d91d-455f-b08f-d514b71c8a29.mov
